provisioning_templates/snippet/kickstart_networking_setup.erb

```diff
@@ -135,27 +142,34 @@ EOF
 <% dhcp = subnet.dhcp_boot_mode? -%>
 
 # <%= interface.identifier %> interface
-real=`ip -o link | grep <%= interface.inheriting_mac -%> | awk '{print $2;}' | sed s/:$//`
-<% if virtual -%>
-  real=`echo <%= interface.identifier -%> | sed s/<%= interface.attached_to -%>/$real/`
-<% end -%>
-
-# ifcfg files are ignored by NM if their name contains colons so we convert colons to underscore
+real=<%= interface.identifier %>
 <% if alias_type -%>
 sanitized_real=`echo $real | sed s/:/_/`
 <% else -%>
 sanitized_real=$real
 <% end -%>
 
+cat << EOF >> /etc/hosts
```

Comment:
I'm not sure everyone wants this. Maybe it should be optional behind a parameter?